homeassistant/bootstrap.py

```diff
@@ -90,7 +90,7 @@
 from .helpers.typing import ConfigType
 from .setup import (
     BASE_PLATFORMS,
-    DATA_SETUP_STARTED,
+    _setup_started,
```

Comment:
It's unexpected to import a protected function.

Reply:
I was a bit worried we would have another custom component digging into the startup data manually since the last time we had someone do that it was a crash on startup problem when we changed it. 

I'll add a comment here